### DISCUSSION TITLE
Live website translation

### DISCUSSION THREAD

**Discussion by Will9982890** [WeblateOrg/weblate#8144] [Q&A]
I currently maintain a website, straight HTML, that is using the old method of Google Translate embedded in the site, which allows for live translation on the fly through JavaScript.  The Google Translate code still works, but it is outdated, slow, and no longer officially officially support so I'm looking for something similar but locally hosted.

I've looked through your documentation and discussions, but there is nothing I can find that allows for this.  I would appreciate someone pointing me in the right direction, if this is even possible.

Thank you in advance.

**Comment by Kentoseth** [comment]
Weblate doesn't appear to be a tool designed for live machine-first translations like Google Translate (AFAICT).

What might work for you that is Open Source is: https://libretranslate.com/

**Comment by nijel** [comment]
You can translate the website in Weblate and use [Automatic translation add-on](https://docs.weblate.org/en/latest/admin/addons.html#automatic-translation) to pre-seed your translations via machine translation service of your choice. That way you can get human fixes into the process as well.
